apps/web/jest.config.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
To rzuca ci jakieś errory?

Reply:
W pierwszej linijce dostaję: 
> Require statement not part of import statement. eslint(@typescript-eslint/no-var-requires)
![image](https://user-images.githubusercontent.com/27455716/221851965-31a3f2c8-1a20-48e4-8890-121c70f0ddfc.png)


może wyłączyć tę regułę w `.eslintrc.json`?

Reply:
Nie spoko. Może być